src/System.Management.Automation/engine/SpecialVariables.cs

```diff
@@ -255,6 +255,11 @@ internal static class SpecialVariables
 
         internal static readonly VariablePath InformationPreferenceVarPath = new VariablePath(InformationPreference);
 
+        internal const string PSNativeCommandUseErrorActionPreference = nameof(PSNativeCommandUseErrorActionPreference);
+
+        internal static readonly VariablePath PSNativeCommandUseErrorActionPreferenceVarPath =
+            new VariablePath(PSNativeCommandUseErrorActionPreference);
```

Comment:
```suggestion
        internal static readonly VariablePath PSNativeCommandUseErrorActionPreferenceVarPath = new(PSNativeCommandUseErrorActionPreference);
```
You don't need make this change. I just think this would be a good fit for the `new()` syntax :) (static field)